tests/test_browser.py

```diff
@@ -2584,10 +2584,19 @@ def test_webgl_shader_source_length(self):
       self.btest(path_from_root('tests', 'webgl_shader_source_length.cpp'), args=opts + ['-lGL'], expected='0', timeout=20)
 
   def test_webgl2(self):
-    for opts in [[], ['-O2', '-g1', '--closure', '1', '-s', 'WORKAROUND_OLD_WEBGL_UNIFORM_UPLOAD_IGNORED_OFFSET_BUG=1'], ['-s', 'FULL_ES2=1']]:
+    for opts in [
+      [],
+      ['-O2', '-g1', '--closure', '1', '-s', 'WORKAROUND_OLD_WEBGL_UNIFORM_UPLOAD_IGNORED_OFFSET_BUG=1'],
+      ['-s', 'FULL_ES2=1'],
+    ]:
       print(opts)
       self.btest(path_from_root('tests', 'webgl2.cpp'), args=['-s', 'USE_WEBGL2=1', '-lGL'] + opts, expected='0')
 
+  @requires_graphics_hardware
+  @requires_threads
+  def test_webgl2_pthreads(self):
+    self.btest(path_from_root('tests', 'webgl2.cpp'), args=['-s', 'USE_WEBGL2=1', '-lGL', '-s', 'USE_PTHREADS=1'], expected='0')
```

Comment:
I was a little confused for a moment about why this is testing a file that doesn't use threads with USE_PTHREADS. I assume it is to have USE_PTHREADS but not OFFSCREEN_FRAMEBUFFER set above; could you add a small comment about that?

Reply:
Adding, thanks.